foodnet/membership/models.py

```diff
@@ -63,39 +67,43 @@ def __str__(self):
 
 class Member(models.Model):
```

Comment:
It looks like it was modelled that way (see the tests, the comment `# Member is a set of users (user profiles)` is explanatory).
But then, it looks like a user at sign up will have to select:
`Department -> Membership -> Member`
which I find frankly a bit too much, wasn't the idea that a single `Membership` holds multiple `User`, and that's it? I mean, what are the reason to have the `Member` indirection?